ubuntu/debian/libignition-rendering-core-dev.install

```diff
@@ -1,12 +1,9 @@
-usr/include/ignition/rendering[0-99]/ignition/rendering/*.hh
-usr/include/ignition/rendering[0-99]/ignition/rendering/base/*.hh
-usr/include/ignition/rendering[0-99]/ignition/rendering.hh
-usr/include/ignition/rendering[0-99]/gz/rendering/*.hh
-usr/include/ignition/rendering[0-99]/gz/rendering/base/*.hh
-usr/include/ignition/rendering[0-99]/gz/rendering/detail/*.hh
-usr/include/ignition/rendering[0-99]/gz/rendering.hh
-usr/lib/*/libignition-rendering[0-99].so
-usr/lib/*/pkgconfig/ignition-rendering[0-99].pc
-usr/lib/*/cmake/ignition-rendering[0-99]/ignition-rendering*
-usr/share/ignition/ignition-rendering[0-99]/ignition-rendering*.tag.xml
-usr/share/ignition/ignition-rendering[0-99]/media/*
+usr/include/*/rendering[0-99]/*/rendering/*.hh
+usr/include/*/rendering[0-99]/*/rendering/base/*.hh
+usr/include/*/rendering[0-99]/*/rendering.hh
+usr/include/*/rendering[0-99]/gz/rendering/detail/*.hh
+usr/lib/*/lib*-rendering[0-99].so
+usr/lib/*/pkgconfig/*-rendering[0-99].pc
+usr/lib/*/cmake/*-rendering[0-99]/*-rendering*
+usr/share/ignition/*-rendering[0-99]/*-rendering*.tag.xml
+usr/share/ignition/*-rendering[0-99]/media/*
```

Comment:
I think these last 2 can safely do `s/ignition/*/`

The same for all other `share` in this package